Scripts/UI/Base/UserInterface.gd

```diff
@@ -0,0 +1,103 @@
+extends Control
+class_name UserInterface
+## UserInterface Class: [br]
+## A Base UI class to contain common UI features within the game such as:[br]
+## - Controller Ui Support
+## - viewport Text Reizing (currently requires text size override to be on.).[br]
+## - Translation Switching (Unimplemented).
+
+@export_group("Focus Start", "start")
+
+enum FOCUS_START_ENUM 
+{
+	## Activates when ready function is called.
+	ready,
```

Comment:
Generally put enum members in MACRO_CASE.

Reply:
Fixed